Formula/bicep.rb

```diff
@@ -0,0 +1,15 @@
+class Bicep < Formula
+  desc "Bicep: next generation template language for Azure Resource Manager (ARM)"
+  homepage "https://github.com/Azure/bicep"
+  url "https://github.com/Azure/bicep/releases/download/v0.1.37-alpha/bicep-osx-x64"
+  sha256 "9502397b2680db391135d9558efefb1e14c3b1c4cd6385d8b53e50bf49b2dcbf"
```

Comment:
Awesome! Presumably once we've released a new version, we just need to make sure we come back and update this file accordingly. Do you know there are any options to automate this?

Reply:
Can it point to latest (`.../bicep/releases/latest/download/...`)? Or is that not a good practice?

Reply:
That'll cause a mismatch in the SHA256 hash when the new release is generated

Reply:
Yep, this should be updated each time there is a release. We don't want to point to latest because people will want to know (and possibly pin) the version they have installed.

We should definitely add this to any release automation that we have though (where is that automation currently?)

Reply:
We use GitHub actions right now. The build workflow we use for both merge validation and release builds is at https://github.com/Azure/bicep/blob/master/.github/workflows/build.yml. The easiest way is probably to add a new job that's similar to `build-vsix` or `build-windows-setup`.

Reply:
We use NerdBank.GitVersioning to determine the version also.